tidb-cloud/manage-user-access.md

```diff
@@ -119,7 +88,63 @@ To invite a member to an organization, perform the following steps:
 >
 > The verification link in the email will expire in 24 hours. If your user doesn't receive the email, click **Resend**.
 
-## Invite a project member
+### Remove an organization member
+
+If you are the owner of an organization, you can remove organization members. Otherwise, skip this section.
+
+To remove a member from an organization, perform the following steps:
+
+> **Note:**
+>
+> If a member is removed from an organization, the member is removed from the belonged projects either.
+
+1. Click <MDSvgIcon name="icon-top-organization" /> **Organization** in the upper-right corner of the TiDB Cloud console.
+
+2. Click **Organization Settings**. The organization settings page is displayed.
+
+3. Click **User Management**, and then select the **By All Users** tab.
+
+4. Click **Delete** in the user row that you want to delete.
+
+## Manage project access
```

Comment:
According to the following sections, this title seems more appropriate to be `Manage projects`, not access to them.